SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+Security is important for us. Security vulnerabilities or suspected security vulnerabilities should be reported to the project's maintainers privately, to minimize attacks against current users of Kroxylicious before they are fixed.
+
+Reported vulnerabilities will be investigated and patched on the next patch (or minor) release as soon as possible.
+
+IMPORTANT: Please do not file public issues on GitHub for security vulnerabilities
+
+To report a vulnerability or a security-related issue, please email the private mailing list kroxylicious-admins at redhat dot com with the details of the vulnerability.
```

Comment:
Really makes me want to figure out non RH mail services for the project but I don't think the ROI is really there.